core/src/main/java/com/datastax/oss/driver/internal/core/type/codec/registry/CachingCodecRegistry.java

```diff
@@ -371,6 +372,23 @@ protected GenericType<?> inspectType(@NonNull Object value, @Nullable DataType c
             inspectType(firstValue, cqlType == null ? null : ((MapType) cqlType).getValueType());
         return GenericType.mapOf(keyType, valueType);
       }
+    } else if (value instanceof CqlVector) {
+      CqlVector<?> set = (CqlVector<?>) value;
```

Comment:
nit: should this variable be `vector`?

Reply:
Sure, that prolly makes more sense.  Flippin' copy/paste... 😦

Reply:
Fixed.